samples/mpsl/timeslot/boards/nrf5340pdk_nrf5340_cpunet.overlay

```diff
@@ -0,0 +1,11 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+
+&uart0 {
```

Comment:
Can we use this overlay for all cases of `NCS_SAMPLE_EMPTY_APP_CORE_CHILD_IMAGE`?

Reply:
Good suggestion. Not sure how easy that is. @tejlmand, ideas?